src/Interface/Modules/Visualization/CreateStandardColorMapDialog.cc

```diff
@@ -71,13 +71,13 @@ CreateStandardColorMapDialog::CreateStandardColorMapDialog(const std::string& na
   connect(invertCheck_, SIGNAL(toggled(bool)), this, SLOT(onInvertCheck(bool)));
 
   auto defaultMap = StandardColorMapFactory::create();
-  auto rainboxIndex = colorMapNameComboBox_->findText("Rainbow", Qt::MatchExactly);
-  if (rainboxIndex >= 0)
+  auto turboIndex = colorMapNameComboBox_->findText("Turbo", Qt::MatchExactly);
```

Comment:
Discussed with @jessdtate, let's keep the default SCIRun's Rainbow.

Reply:
switch back to rainbow

Reply:
but the turbo has purple!!

Reply:
and yellow!!

Reply:
So are we doing a vote here or should I go ahead a change it back?